### PR TITLE
Rename staff to manager

### PR DESCRIPTION
**Pull request by PFischbeck**
This changes `staff` to `manager`, which hopefully makes #1109 (staff mode) easier.

### CODE REVIEW

File: evap/rewards/views.py [hunk 1]
```diff
@@ -1,127 +1,127 @@
...
+
+
+@manager_required
+def semester_activation(request, semester_id, active):
+    semester = get_object_or_404(Semester, id=semester_id)
+    active = active == 'on'
+
+    SemesterActivation.objects.update_or_create(semester=semester, defaults={'is_active': active})
+
+    return semester_view(request=request, semester_id=semester_id)
```
Comment (karyon) [inline evap/rewards/views.py]: line endings got screwed up

File: evap/evaluation/auth.py [hunk 2]
```diff
@@ -136,6 +136,15 @@ def check_user(user):
     return user_passes_test(check_user)(view_func)
```
Comment (karyon) [inline evap/evaluation/auth.py]: is `staff_required` used anywhere after applying this PR?

File: evap/evaluation/migrations/0079_rename_staff_to_manager.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.0.7 on 2018-07-30 19:25
+
+from django.db import migrations, models
+
+
+def rename_group(apps, schema_editor, old_name, new_name):
+    Group = apps.get_model('auth', 'group')
+    db_alias = schema_editor.connection.alias
+    Group.objects.using(db_alias).filter(name=old_name).update(name=new_name)
```
Comment (karyon) [inline evap/evaluation/migrations/0079_rename_staff_to_manager.py]: why is this `using` part necessary?

File: evap/rewards/views.py (hunk 1 above)
Comment (PFischbeck) [reply evap/rewards/views.py]: Fixed

File: evap/evaluation/auth.py (hunk 2 above)
Comment (PFischbeck) [reply evap/evaluation/auth.py]: This is not used yet, so I removed it.

File: evap/evaluation/migrations/0079_rename_staff_to_manager.py (hunk 3 above)
Comment (PFischbeck) [reply evap/evaluation/migrations/0079_rename_staff_to_manager.py]: This is the way it is suggested in the official documentation
https://docs.djangoproject.com/en/2.0/ref/migration-operations/#runpython

**Comment by PFischbeck** [comment]
@karyon Ready for another review. The Codacy checks are false positives, as far as I can tell.

**Comment by karyon** [comment]
i can take over any merge conflicts in other PRs.

**Pull request merged**